mantidproject/mantid
# Engg:: Save focused data in 3 formats

**This should be merged after the following issue has been merged #13903**
- [x] #13903 merged

_Fixes #13854 & #13783 & #13853_ 

**To Test**
#13854

Click the `Output Files` check-box which should save the generated files/focused workspaces in to `OpenGenie`, `GSS` & `FocusedXYE` formats with appropriate labels. 

 #13783
Saves File As:
- `ENGINX_...<RUN_NUMBER>_..._bank_1` .his, .gss, .dat, etc. for banks 1, 2, etc.
#13853
- In the GUI use the MWRun widget instead of raw numbers
